docs/platforms/javascript/common/options.mdx

```diff
@@ -0,0 +1,454 @@
+---
+title: SDK Options
+description: "Learn more about how to configure the SDK. These options are set when the SDK is first initialized, passed to the init function as an object."
+customCanonicalTag: "/platforms/javascript/options/"
+sidebar_order: 2
+---
+
+<PlatformContent includePath="configuration/config-intro" />
+
+## Available Options
+
+<SdkOptionsOverview />
+
+## Core Options
+
+<SdkOption name='dsn' type='string' envVar='SENTRY_DSN'>
+The _DSN_ tells the SDK where to send the events. If this value is not provided, the SDK will try to read it from the `SENTRY_DSN` environment variable. If that variable also does not exist, the SDK will just not send any events.
+
+In runtimes without a process environment (such as the browser) that fallback does not apply.
+
+Learn more about [DSN utilization](/product/sentry-basics/dsn-explainer/#dsn-utilization).
+
+</SdkOption>
+
+<SdkOption name="debug" type='boolean' defaultValue='false'>
+
+Turns debug mode on or off. If debug is enabled SDK will attempt to print out useful debugging information if something goes wrong with sending the event. The default is always `false`. It's generally not recommended to turn it on in production, though turning `debug` mode on will not cause any safety concerns.
```

Comment:
safety sounds weird here 🤔 either we say "security" or write something like it won't expose any critical information? Or just omit it?

```suggestion
Turns debug mode on or off. If debug is enabled SDK will attempt to print out useful debugging information if something goes wrong with sending the event. The default is always `false`. It's generally not recommended to turn it on in production, though turning `debug` mode on will not cause any security concerns.
```

Reply:
maybe we should just remove the last sentence here, not sure how relevant this is actually..?